kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/kommunicate/services/KmService.java

```diff
@@ -148,4 +149,25 @@ public void onFailure(String response, Exception e, Context context) {
     public static void updateChannel(Context context, GroupInfoUpdate groupInfoUpdate, AlChannelUpdateTask.AlChannelUpdateListener listener) {
         new AlChannelUpdateTask(context, groupInfoUpdate, listener).execute();
     }
+
+    /**
+     * the wrapper method to get feedback for given conversation
+     * @param conversationId the groupId of the conversation
+     * @return the response object, response.getData() will return null in case of feedback not found
+     */
+    public synchronized String getConversationFeedback(String conversationId) {
+        String response = clientService.getConversationFeedback(conversationId);
```

Comment:
response variable is not required, can directly return

Reply:
done